src/main/config/messages_template.xml

```diff
@@ -413,6 +413,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ082E" type="ERROR">
+    <reason>Resource %1 is referenced with incorrect path capitalization</reason>
```

Comment:
Is the capitalization mismatch in the path, the filename, or either one? 
Will `%1` be expanded to the full path, or just the filename?

Error messages are better when they're specific, rather than re-using the same message for different cases. 
If we can't separate the causes (or we're not sure where the problem lies), this might be better:

> The resource referenced as %1 is capitalized differently on disk

Can we throw separate errors for strict vs. lax processing? 

Good error messages emphasize the solution and how to complete the action, so ideally, we'd add more info, like:

> Adjust the reference or filename to correct the case.

_or_

> The reference will be rewritten to match the file system case.

Reply:
`%1` will expand to absolute path. The log message in different processing modes:

- strict: `[DOTJ082E][ERROR] Resource file:/Volumes/tmp/src/Topic.dita is referenced with incorrect path capitalization`
- skip: `[DOTJ082E][ERROR] Resource file:/Volumes/tmp/src/Topic.dita is referenced with incorrect path capitalization, using invalid value.`
- lax: `[DOTJ082E][ERROR] Resource file:/Volumes/tmp/src/Topic.dita is referenced with incorrect path capitalization, using file:/Volumes/tmp/src/topic.dita value.`

Reply:
My preference is to use the same error code regardless of the processing mode, because the actual failure/error is the same in all of them. If our message format supported different messages per processing mode, then we could have different messages. Right now the `, using invalid value.` comes from the Java code. It's not pretty, but gets the job done.